shopfloor/data/ir_cron_data.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="ir_cron_autovacuum_shopfloor_log" model="ir.cron">
+        <field name="name">AutoVacuum Shopfloor Logs</field>
```

Comment:
A memory of Éric Caudal which is still in my head! https://github.com/OCA/server-tools/pull/360#discussion_r52842317
Will you honor him by fixing this label? :)

Reply:
Done